code/game/machinery/computer/launchpad_control.dm

```diff
@@ -92,6 +200,7 @@
 	if(.)
 		return
 	var/obj/machinery/launchpad/current_pad = launchpads[selected_id]
+	var/checks
```

Comment:
You don't need this upvalue. Just do `var/checks = teleport_checks(current_pad)` where you actually use it.